pkgs/tools/system/tuptime/default.nix

```diff
@@ -20,9 +20,14 @@ stdenv.mkDerivation rec {
     mkdir -p $out/share/man/man1
     cp src/man/tuptime.1 $out/share/man/man1/
 
-    # upstream only ships this, there are more scripts there...
-    mkdir -p $out/usr/share/doc/tuptime/contrib
-    cp misc/scripts/uptimed-to-tuptime.py $out/usr/share/doc/tuptime/contrib/
+    install -Dm 0755 $src/misc/scripts/db-tuptime-migrate-4.0-to-5.0.sh $out/share/tuptime/db-tuptime-migrate-4.0-to-5.0.sh
+
+    # no _tuptime group is created by the nixos module, only change the user
```

Comment:
Why not create the group too?

Reply:
it seems like it's only setting group because usually the user is created with `useradd` without `-N`, i don't think the group is required

creating the group does seem like the neater solution
i hadn't done that before and i'm not sure of its impact

Reply:
created the group as it removes the need for the patch

Reply:
Sounds good, I think creating a group has no negative consequences and seems more maintainable than the patch :+1: